opennms-doc/guide-admin/src/asciidoc/text/telemetryd/introduction.adoc

```diff
@@ -10,7 +10,7 @@ With _telemetryd_, operators can define listeners supporting different protocols
 image::telemetryd/telemetryd-overview.png[]
 
 The configuration is split in two parts.
-_Listeners_ and attached _Parsers_ on one side, receive telemetry data transported over a specific protocol and parse the protocol according to its specification.
+_Listeners_ and attached _Parsers_ or _Connectors_ on one side, receive telemetry data transported over a specific protocol and parse the protocol according to its specification.
```

Comment:
```suggestion
_Listeners_ and attached _Parsers_ or _Connectors_ on one side receive telemetry data transported over a specific protocol and parse the data according to the protocol's specification.
```

I believe it is the data that is parsed, not the protocol, hence my suggested edit.